src/app/features/activity-list/components/tab-wrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import { Box } from 'leather-styles/jsx';
+
+interface TabWrapperProps {
+  children: React.ReactNode;
+  padContent?: boolean;
+}
+
+export function TabWrapper({ children, padContent = false }: TabWrapperProps) {
```

Comment:
Wonder if this name should be more specific to activity list? Will we use it elsewhere (presume not as in this folder)

Reply:
Yes, good idea. I've updated it. 

More generally we should add our own `Tab` wrapper to the UI library and make sure all tabs always have a set minimum height regardless of content